### PR TITLE
LPS-186643 | [CP 2.0] Unit test

### PR DESCRIPTION
**Pull request by gabrielargomes**
https://issues.liferay.com/browse/LPS-186643

### CODE REVIEW

**Comment by gabrielargomes** [comment]
ci:test:sf

**Comment by pedroliveirav** [comment]
ci:test:relevant

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		cleanup();
+		vi.clearAllTimers();
+		vi.restoreAllMocks();
+	});
+
+	afterAll(() => {
+		vi.useRealTimers();
+	});
+
+	it(`Renders`, async () => {
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx:86]:
```suggestion
	it('renders', async () => {
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+									},
+								],
+							},
+						],
+					},
+				},
+			})),
+		};
+
+		const {debug} = render(
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx:110]:
```suggestion
		const {asFragment} = render(
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+					/>
+				</MockedProvider>
+			</MemoryRouter>
+		);
+
+		await act(async () => {
+			vi.runAllTimers();
+		});
+
+		debug();
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx:132]:
```suggestion
		expect(asFragment()).toMatchSnapshot();
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx (hunk 1 above)
Comment (gabrielargomes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx]: > 

resolved

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+						dxpVersion=""
+						errors={{}}
+						handlePage={() => {}}
+						leftButton=""
+						listType=""
+						project={{}}
+						setFieldValue={() => {}}
+						setFormAlreadySubmitted={() => {}}
+						subscriptionGroupId=""
+						touched={{}}
```
[the remark this answers is not in the document]
Comment (gabrielargomes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/SetupDXPCloudForm.test.jsx]: resolved

**Comment by gabrielargomes** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Comment by gabrielargomes** [comment]
ci:test:relevant

**Comment by gabrielargomes** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**